apps/site/src/app/(home)/sections/About/About.tsx

```diff
@@ -0,0 +1,67 @@
+import Image from "next/image";
+import Lantern from "@/lib/components/Lantern/Lantern";
+import koiJump from "@/assets/images/koi-jump.png";
+import styles from "./About.module.css";
+
+const About = () => {
+	return (
+		<>
+			<section className="container md mx-auto text-center my-14 w-96 sm:w-1/2 xl:w-2/5 relative">
```

Comment:
Issue: `w-96` is too wide for smaller phones (24rem > 320px). The Bootstrap approach I mentioned before relies on max-widths for each breakpoint.